img/info.py

```diff
@@ -35,7 +35,7 @@
 
 extra_data = [
     ";FLAVOR:Marlin",
-    f";TIME:{time_s:.2f}",
```

Comment:
Does dropping the formatter still make it work on the screen? The output generated by creality slicer has 2 decimal points

Reply:
This is one thing i cannot test myself; i don't run stock firmware anymore

Reply:
Yeah, cura generates this output:

```;FLAVOR:Marlin
;TIME:9994
;Filament used: 9.3909m
;Layer height: 0.2
;MINX:66.534
;MINY:68.118
;MINZ:0.2
;MAXX:153.365
;MAXY:151.896
;MAXZ:70.6
;TARGET_MACHINE.NAME:Creality Ender-3 V3 SE
;Generated with Cura_SteamEngine 5.6.0```

and this is what I was looking for. If creality does that with decimal points we can withdraw this last commit of mine, probably both could work